packages/common/component-utils.js

```diff
@@ -0,0 +1,11 @@
+import _ from 'lodash'
+
+export function addBooleanParameter (controller, parameterName) {
+  if (_.has(controller.$attrs, parameterName) && _.isEmpty(controller.$attrs[parameterName])) {
```

Comment:
if attr value is ```false``` it will be considered as empty

Reply:
Changed